projects/web-wallet/rpc-protocol.md

```diff
@@ -641,3 +735,25 @@ An example from metamask we can take inspiration from:
 <img width="336" alt="image" src="https://github.com/HathorNetwork/rfcs/assets/3586068/617d95b3-43ea-4db2-9698-2b517c9d0f5d">
 
 If the request is `first_empty`, `full_path` or `index`, the wallet should ask the user to accept the address being sent
+
+### 3.8 `htr_sendNanoContractTx`
+
+When the request is received, we will validate if the parameters are valid,
+enforcing the data types and rules described in the Reference-Level explanation
+
+If the validation fails, we will respond with a `INVALID_PARAMETERS` error code
+
+If the validation is successful, we will display a confirmation screen,
+with all outputs for manual user validation (UX is yet to be discussed). If the
+user rejects the transaction, we will respond with a `REQUEST_REJECTED` error
+
+The address will default to the address that is registered with the nano
+contract on the wallet or if it's not yet registered, it will default to the
+address with index 0.
+
+The user will be allowed to select any other address from his wallet or manually
```

Comment:
This address is the address that will be used to sign the nano contract transaction, so the user can't manually input any valid address he wishes, he must choose one address from his wallet.

Reply:
Done in a5a5a07177b955da5df021a3d2e144368c61a434